subcommands/devices/cmd.go

```diff
@@ -27,12 +27,36 @@ var configCmd = &cobra.Command{
 }
 
 var updatesCmd = &cobra.Command{
-	Use:   "updates",
-	Short: "Device update history",
+	Use:   "updates <device> [<update-id>]",
+	Short: "Show updates performed on a device",
+	Args:  cobra.RangeArgs(1, 2),
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 1 {
+			doListUpdates(cmd, args)
+		} else {
+			doShowUpdate(cmd, args)
+		}
+	},
+	Example: `
+# List all updates performed on a device:
+fioctl devices updates <device1>
+
+# List the last 2 updates:
+fioctl devices updates <device> -n2
+
+# Show the details of an update:
+fioctl devices updates <device> <update-id>
```

Comment:
From a compatibility perspective it'd be better to have `fioctl devices updates show <device> <update-id>`, as there is a small chance some angry user named their device as "list" or "show" and things will start breaking for them.
Though honestly that's more a funny comment than a practical one.
In general if we don't plan to add any sub-commands - this seems good.

Reply:
there's nothing you can do to an update-event except view so subcommands for this just aren't useful.